src/Schema/Directives/LikeDirective.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Nuwave\Lighthouse\Schema\Directives;
+
+use Nuwave\Lighthouse\Support\Contracts\ArgBuilderDirective;
+
+class LikeDirective extends BaseDirective implements ArgBuilderDirective
+{
+    public static function definition(): string
+    {
+        return /** @lang GraphQL */ <<<'GRAPHQL'
+enum LikePercentageLocation {
+    START
+    END
+    BOTH
+}
+
+"""
+Uses a LIKE query.
+"""
+directive @like(
+  """
+  Specify the database column to compare.
+  Only required if database column has a different name than the attribute in your schema.
+  """
+  key: String
+
+  """
+  Specify the positions of the % in the LIKE comparison. The default is BOTH.
+  """
+  percentage: LikePercentageLocation
```

Comment:
The name `percentage` indicates that the value of this argument would be a numeric value between `0` and `100`.

Reply:
I have no better idea for a name, it seemed clear enough to me. Any suggestions?

Reply:
Yeah, you are right. Bad naming on my part.

What do you say to `wildcard` and `LikeWildcardLocation`

Reply:
How about `SearchMethod`: `(ENDS, STARTS, CONTAINS)` (`%text`, `text%` and `%text%` respectively)?

Reply:
My proposed solution is https://github.com/nuwave/lighthouse/pull/1645#discussion_r549349245